tests/containers/account/awards/AccountAwardsContainer-test.jsx

```diff
@@ -70,6 +70,7 @@ describe('AccountAwardsContainer', () => {
                 contracts: 2,
                 direct_payments: 0,
                 grants: 0,
+                idv: 0,
```

Comment:
This should be removed because this story does not include updates to the account profile page.